app/src/main/java/hr/foi/air/servicesync/ui/components/ReviewCard.kt

```diff
@@ -0,0 +1,104 @@
+package hr.foi.air.servicesync.ui.components
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Star
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import hr.foi.air.servicesync.R
+import hr.foi.air.servicesync.data.Review
+
+@Composable
+fun ReviewCard(
+    review: Review
+) {
+    Card(
+        shape = RoundedCornerShape(8.dp),
+        colors = CardDefaults.cardColors(
+            containerColor = MaterialTheme.colorScheme.surfaceVariant,
```

Comment:
This should use colors from Colors.kt. Change to the color palette of Figma if not the same hex code

Reply:
Sure, but then the rest of the screen is in other color. We need to coordinate those colors better.